lib/build/bundle.js

```diff
@@ -0,0 +1,63 @@
+var assign = require("lodash").assign;
+var minify = require("../stream/minify");
+var devBundle = require("../stream/dev_bundle");
```

Comment:
I think this should be renamed to `makeBundleStream` or something like that. The "dev" part of this feature should be pushed up into the cli.  The `--dev` flag is a special flag that expands to `filter: node_modules/**/*` or whatever we decide on.

That's the *only* place the word *dev* should be, in my opinion. This feature should be useable for creating other types of bundles, like bundles of just your own dependencies, perhaps.